.travis.yml

```diff
@@ -17,6 +16,9 @@ before_install:
   - "export DISPLAY=:99.0"
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
+install:
```

Comment:
well, I think npm 2.x is hosed on node 4. I always run into things like [this](https://github.com/npm/npm/issues/6551) and upgrading npm fixes it. But I guess if you aren't running into trouble yet then there's no reason to upgrade npm. Just beware of it if you hit any odd npm behavior.